README.md

```diff
@@ -1345,6 +1348,21 @@ Installs [`mod_expires`][] and uses the `expires.conf.erb` template to generate
 - `expires_default`: Default algorithm for calculating expiration time using [`ExpiresByType`][] syntax or [interval syntax][]. Default: undef.
 - `expires_by_type`: Describes a set of [MIME `content-type`][] and their expiration times. Valid options: An [array][] of [Hashes][Hash], with each Hash's key a valid MIME `content-type` (i.e. 'text/json') and its value following valid [interval syntax][]. Default: undef.
 
+##### Class: `apache::mod::ext_filter`
+
+Installs and configures [`mod_ext_filter`][].
+
+~~~ puppet
+class{'apache::mod::ext_filter':
+  ext_filter_define => { 'slowdown'       => 'mode=output cmd=/bin/cat preservescontentlength',
+                         'puppetdb-strip' => 'mode=output outtype=application/json cmd="pdb-resource-filter"'},
+}
```

Comment:
i'd rather write this as:
~~~ puppet
class { 'apache::mod::ext_filter':
  ext_filter_define => {
    'slowdown'       => 'mode=output cmd=/bin/cat preservescontentlength',
    'puppetdb-strip' => 'mode=output outtype=application/json cmd="pdb-resource-filter"',
  },
}
~~~